src/components/search/facets/selected-filters.vue

```diff
@@ -55,13 +54,7 @@
       v-if="$x.device === 'desktop'"
       v-slot="{ selectedFilters }"
       data-test="clear-filters-toolbar"
-      class="
-        x-button-sm x-button-lead x-button-outlined
-        x-border-radius--20
-        x-padding--05 x-padding--top-03 x-padding--bottom-03
-        x-margin--left-03
-        x-list__item--flex-none
-      "
+      class="x-button-sm x-button-lead x-button-outlined x-border-radius--20"
```

Comment:
Removing `x-list__item--flex-none` may cause the `Clear filters` button to wrap the text when there are many selected items. But it is maybe a corner case, I'm fine if we don't add the class.

![image](https://user-images.githubusercontent.com/77450928/202400771-d6588a16-60bf-4886-9ca7-4ba37ed63b0a.png)